spec/support/models.rb

```diff
@@ -43,7 +43,7 @@ def define_columns(t, array: false)
 
   if t.is_a?(ActiveRecord::TypedStore::DSL)
     t.integer :grades, array: true
-    t.string :tags, array: true, null: false, default: [].to_yaml
```

Comment:
Wondering if there is a rational why we provided a non-array default to this field? Is it for test that the provided default is wrong and to fallback to `[]`? 

Happy to not re-use this field and to create another one with a proper default.